src/web/mjs/connectors/MangaCanBlog.mjs

```diff
@@ -30,20 +30,25 @@ export default class MangaCanBlog extends Connector {
 
     async _getChapters(manga) {
         let request = new Request(this.url + manga.id, this.requestOptions);
-        let data = await this.fetchDOM( request, 'div#latestchapters table tr td a.chaptersrec' );
+        let data = await this.fetchDOM( request, 'div#chapterlist ul li div.eph-num a' );
```

Comment:
Looks like `WordPressMangastream`, maybe it is possible to use template?

Reply:
Damn, you're right ! We need a template detector (i don't create much connector, only fixing and therefore have no idea when i encounter one)